Assets/Code/src/Runtime/Match/Systems/PhysicsMoveSystem.cs

```diff
@@ -9,61 +9,133 @@
 using Unity.Physics.Systems;
 using Unity.Mathematics;
 
-public class PhysicsMoveSystem : SystemBase {
+public class PhysicsMoveSystem : SystemBase
+{
 
-    protected override void OnUpdate() {
+    protected override void OnUpdate()
+    {
         float deltaTime = Time.fixedDeltaTime;
         float curInput = (Keyboard.current.aKey.isPressed) ? -1 : (Keyboard.current.dKey.isPressed) ? 1 : 0;
 
         Entities
         .WithoutBurst()
-        .ForEach((ref PhysicsMoveData moveData, ref PhysicsVelocity vel) => {
+        .ForEach((ref PhysicsMoveData moveData, ref PhysicsVelocity vel, in GroundedTrigger grounded) =>
+        {
             float3 newVel = vel.Linear;
 
+            //resetting the number of air jumps when the grouded trigger is triggered
+            if (grounded.triggered)
+            {
```

Comment:
Here and elsewhere, please adhere to the project code style and use K&R bracket style.